pkg/crc/services/dns/template.go

```diff
@@ -0,0 +1,166 @@
+package dns
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/services"
+)
+
+const (
+	coreFileTemplate = `{{ .Domain }}:{{ .Port }} {
+    log
+    errors
+    file {{ .ZonefilePath }}
+}`
+	zoneFileSOATemplate = `$ORIGIN .
+@    3600   IN      SOA     ns.{{ .Domain }}. root.({{ .Domain }}). (
+                              5         ; serial
+                         604800         ; refresh
+                          86400         ; retry
+                        2419200         ; expire
+                          36000         ; minimum
+			  )
+`
+	zoneFileAddressTemplate = `; Name Server Information
+_etcd-server-ssl._tcp.{{ .ClusterName}}.{{ .BaseDomain }}. IN SRV 10 10 2380 etcd-0.{{ .ClusterName }}.{{ .BaseDomain }}.
+{{ .Hostname }}.{{ .ClusterName }}       IN  A     {{ .IP }}            ;
+api.{{ .ClusterName }}                      IN  A     {{ .IP }}            ;
+api-int.{{ .ClusterName }}                  IN  A     {{ .IP }}            ;
+etcd-0.{{ .ClusterName }}                   IN  A     {{ .IP }}            ;
+*.{{ .AppsDomain }}.          IN  A     {{ .IP }}            ;
```

Comment:
do we really need `;` at the end of those fqdn entry side?

Reply:
these are comments, not essential but part of the zonefile definition. I can remove them...
I am more concerned about the fact FQDNs are needed instead :-/